sdk/minio/src/helpers/client.ts

```diff
@@ -0,0 +1,45 @@
+import type { Client } from "minio";
+import { createServerMinioClient } from "../minio.js";
+import { ensureServer } from "./runtime.js";
+
+let cachedClient: Client | null = null;
+
+/**
+ * Gets or creates a MinIO client instance
+ *
+ * @returns A MinIO client instance
+ * @throws Will throw an error if not called on the server or if environment variables are missing
+ *
+ * @example
+ * import { getMinioClient } from "@settlemint/sdk-minio";
+ *
+ * const client = await getMinioClient();
+ * const buckets = await client.listBuckets();
+ */
+export async function getMinioClient(): Promise<Client> {
+  ensureServer();
+
+  if (cachedClient) {
+    return cachedClient;
+  }
+
+  // Check for required environment variables
+  const instance = process.env.SETTLEMINT_MINIO_ENDPOINT;
+  const accessKey = process.env.SETTLEMINT_MINIO_ACCESS_KEY;
+  const secretKey = process.env.SETTLEMINT_MINIO_SECRET_KEY;
+
+  if (!instance || !accessKey || !secretKey) {
```

Comment:
would remove this, the schema validation is handling this already in 'createServerMinioClient'